#!/usr/bin/env python
PACKAGE = "rope_recog"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_ulimit", int_t, 0, "Upper limit of H", 255, 0, 255)
gen.add("h_llimit", int_t, 0, "Lower limit of H", 0, 0, 255)
gen.add("s_ulimit", int_t, 0, "Upper limit of S", 255, 0, 255)
gen.add("s_llimit", int_t, 0, "Lower limit of S", 0, 0, 255)
gen.add("v_ulimit", int_t, 0, "Upper limit of V", 255, 0, 255)
gen.add("v_llimit", int_t, 0, "Lower limit of V", 0, 0, 255)

exit(gen.generate(PACKAGE, "rope_recog", "ColorFilter"))
